### ISSUE TITLE
Plugin not functioning since Twitter API update from v1.1 to v2

### ISSUE THREAD

**Issue by impactpractise** [joedolson/wp-to-twitter#5]
**Describe the bug**
Since twitter's API update to APi v2, the WP to twitter plugin doesn't work anymore. 

» 403 Forbidden: Your request has been received, but rejected by Twitter. (Error Code: 453: You currently have access to a subset of Twitter API v2 endpoints and limited v1.1 endpoints (e.g. media post, oauth) only. If you need access to this endpoint, you may need a different access level. You can learn more here: https://developer.twitter.com/en/portal/product)

We already updated the API in the twitter dashboard. We expect the endpoints wp to twitter sends and receives requests from needs to be updated to v2.

**To Reproduce**
1. Publish a post and tweet it using wp to twitter
2. Go to wordpress backend, wp to twitter
3. Check the error log on top
4. See error // use wp to twitter test

**Expected behavior**
Tweets should be published as expected. For that, the API needs to be updated to v2.

We updated the API in 
**Screenshots**
<img width="1603" alt="Screenshot 2023-07-21 at 15 30 08" src="https://github.com/joedolson/wp-to-twitter/assets/31695597/07834a11-d80b-4efb-aefc-d64489dff806">

**Comment by talgalili** [comment]
I also have this issue.
And many others:
https://wordpress.org/support/topic/403-forbidden-the-request-is-understood-but-has-been-refused-by-twitter/

**Comment by joedolson** [comment]
Fixed in 4.0.0

**Comment by talgalili** [comment]
Hey @joedolson 

I've updated to 4.0, and got the following failure:
```
 Fatal error: Uncaught Error: Call to undefined method RuntimeException::getResponse()
in /home4/pythonbloggers/public_html/wp-content/plugins/wp-to-twitter/wp-to-twitter.php on line 456

Call stack:

wpt_post_to_twitter()
wp-content/plugins/wp-to-twitter/wpt-functions.php:123
wpt_check_functions()
wp-content/plugins/wp-to-twitter/wp-to-twitter-manager.php:194
wpt_updated_settings()
wp-content/plugins/wp-to-twitter/wp-to-twitter-manager.php:212
wpt_update_settings()
wp-includes/class-wp-hook.php:310
WP_Hook::apply_filters()
wp-includes/class-wp-hook.php:334
WP_Hook::do_action()
wp-includes/plugin.php:517
do_action()
wp-admin/admin.php:259
```

Any idea on what to do?
